### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dav1dpgit**
added the instance of Agora in https://modulo.network

happy to change the wording to remove redundant information, like changing it to:

"Lightning payment over tor embedded in 3rd party BTCPay Server:  https://modulo.network"

### CODE REVIEW

**Comment by casey** [comment]
Can you link directly to the agora instance? I couldn't find it at modulo.network

**Comment by dav1dpgit** [comment]
I can provide the tor link if that is what you are asking?  The agora link currently is embedded in the BTCPay Server button, where the pop up purchase button says continue with on chain or copy the tor link to a tor browser for lightning.

**Comment by dav1dpgit** [comment]
Just updated with the Agora tor link--welcome your feedback.

File: README.md [hunk 1]
```diff
@@ -15,6 +15,7 @@ Public Agora instances:
 
 - [tmcalh6tjskfmnjsipi4tqlzcr4ltracuup6gp4vkyi4e5s42ooef2yd.onion](http://tmcalh6tjskfmnjsipi4tqlzcr4ltracuup6gp4vkyi4e5s42ooef2yd.onion): [@yzernik's](https://github.com/yzernik) instance, accessible over Tor
 - A static snapshot of our decommissioned test instance is available [here](https://agora-org.github.io/agora/).
+- Lightning payment over tor embedded in 3rd party BTCPayServer instance:  https://modulo.network after clicking purchase button (http://jzqqbzci3b2jlnvvkuxp2gmfh7if5443ite3ho7wy3phnq7tz4tspbid.onion/files/)
 - Open an issue or submit a PR if you run an Agora instance and would like it to appear in this readme!
```
Comment (casey) [inline README.md:19]:
```suggestion
- [tmcalh6tjskfmnjsipi4tqlzcr4ltracuup6gp4vkyi4e5s42ooef2yd.onion](http://tmcalh6tjskfmnjsipi4tqlzcr4ltracuup6gp4vkyi4e5s42ooef2yd.onion): [@yzernik's](https://github.com/yzernik) instance, accessible over Tor
- [Modulo Network mining spreadsheets](https://modulo.network). [Direct link to Agora instance.](http://jzqqbzci3b2jlnvvkuxp2gmfh7if5443ite3ho7wy3phnq7tz4tspbid.onion/files/).
- A static snapshot of our decommissioned test instance is available [here](https://agora-org.github.io/agora/).
- Open an issue or submit a PR if you run an Agora instance and would like it to appear in this readme!
```

**Comment by casey** [comment]
Looks good! I was just confused where the Agora instance was. I bumped it up the list and added a description.

**Comment by dav1dpgit** [comment]
> Looks good! I was just confused where the Agora instance was. I bumped it up the list and added a description.

OK!

I thought it would be good to mention how people can embed the agora tor link into payment buttons of BTCPS (or any buttons for that matter,) a testament to agora's application versatility.

**Comment by dav1dpgit** [comment]
Is there anything I need to do from here?

**Comment by casey** [comment]
Sorry! The tests failed, which blocked auto-merge, and I didn't notice. It's probably a flakey test, I'll re-run them.

**Pull request merged**